helm/ffc-demo-payment-service-core/templates/deployment.yaml

```diff
@@ -15,7 +17,7 @@ spec:
         environment: {{ quote .Values.environment }}
       annotations:
         redeployOnChange: {{ quote .Values.container.redeployOnChange }}
-    spec:
+    spec:      

```

Comment:
Sneaky whitespace

Reply:
😮 but VSCode removes trailing white-space automatically on save! As I just proved when I removed that white-space by simply opening and saving the file. 😄